File: chicken_and_dog.py
numLeg1d = 4
numLeg1c = 2
dog, chick = 36, 0
while True:
    if (dog + chick == 36) and (dog * numLeg1d + chick * numLeg1c == 100):
        break;
    dog -= 1
    chick +=1
print("So ga:", chick)
print("So cho:", dog)
        
        

File: chicken_and_dog.js
const numLeg1d = 4;
const numLeg1c = 2;
let chick = 0; 
let dog = 36;
while(true)
{
    if((chick + dog == 36) && (chick * numLeg1c + dog * numLeg1d == 100))
    {
        console.log("So ga: ", chick);
        console.log("So cho: ", dog);
        break;
    }
    ++chick;
    --dog;
}